activiti-cloud-starter-runtime-bundle/src/test/java/org/activiti/cloud/starter/tests/helper/ProcessInstanceRestTemplate.java

```diff
@@ -108,12 +112,21 @@ public ProcessInstanceRestTemplate(TestRestTemplate testRestTemplate) {
         return startProcess(PROCESS_INSTANCES_ADMIN_RELATIVE_URL,startProcess);
     }
     
-    private ResponseEntity<CloudProcessInstance> startProcess(String baseURL,StartProcessPayload startProcess) {
-        ResponseEntity<CloudProcessInstance> responseEntity = testRestTemplate.exchange(baseURL,
-                                                                                        HttpMethod.POST,
-                                                                                        new HttpEntity<>(startProcess),
-                                                                                        new ParameterizedTypeReference<CloudProcessInstance>() {
-                                                                                        });
+    public ResponseEntity<CloudProcessInstance> adminStartProcessWithoutCheck(StartProcessPayload startProcess) {
```

Comment:
What does `withoutCheck` means? What is not been checked?

Reply:
This method is not used (was planned at the beginning), I have removed it